src/main/java/hudson/scm/CVSChangeLogSet.java

```diff
@@ -179,23 +177,19 @@ public static CVSChangeLogSet parse(final Run<?, ?> build, RepositoryBrowser<?>
         private static final DateFormat[] dateFormatters = new SimpleDateFormat[]{
             new SimpleDateFormat(CHANGE_DATE_FORMATTER_PATTERN),
             new SimpleDateFormat("yyyy/MM/dd HH:mm:ss")};
-        private static final DateFormat DATE_FORMATTER = new SimpleDateFormat(
-                "yyyy-MM-dd");
-        private static final DateFormat TIME_FORMATTER = new SimpleDateFormat(
-                "HH:mm");
 
         private String user;
         private String msg;
         private final List<File> files = new ArrayList<File>();
-        private Calendar changeDate;
+        private long changeDate;
```

Comment:
Should the field be renamed? IIUC in the current state it may drop the historical data if it is persisted somewhere. With renamed field ti would be an Old Data Monitor instead of crappy error message at least

Reply:
The field is truly private. It is never saved or read via XStream.

Reply:
(It _is_ visible in the serial form for Remoting, of course, but that has no diachronic implications.)